src/contexts/SiteLaunchContext.tsx

```diff
@@ -10,10 +10,11 @@ import { launchSite } from "services/SiteLaunchService"
 
```

Comment:
the design wise, when you check a box it is under the title "I need support". Feels like I am clicking the checkbox for I need support on that task
![Screenshot 2023-08-15 at 11 44 15 PM](https://github.com/isomerpages/isomercms-frontend/assets/5507822/fb877a4b-c24c-4f3f-b395-5b4c394fcdd7)


cc @sehyunidaaa

Reply:
Thanks for bringing this up @harishv7! It's a very fair point, I can see the confusion. I think we can kiv it for now though, because the copy above explains that these are tasks to be completed. Altho it's only one datapoint we also saw little confusion from a PA site launch.

I can keep this on my list though and observe if we see any confusion on session replays for future site launches

Reply:
Sure thanks for taking note of this! Can be a future enhancement